plugins/modules/azure_rm_aduser.py

```diff
@@ -0,0 +1,406 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2020 Cole Neubauer, (@coleneubauer)
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
```

Comment:
please remove the ANSIBLE_METADATA block, it is no longer needed.

Reply:
This block has now been removed